canonical-web-and-design/ubuntu.com
# Redirect /blog back to blog.ubuntu.com, for now

Roll back the /blog release

QA
--

`./run`, go to `/blog`, `/blog/cloud-and-server`, `/blog/canonical-announces-support-for-ubuntu-on-windows-subsystem-for-linux-2` etc. - check you end up on the right page on blog.ubuntu.com each time
